spring/boot-autoconfigure/src/test/resources/config/application-test_reactive.yml

```diff
@@ -0,0 +1,17 @@
+server:
+  address: 127.0.0.1
+  ssl:
+    # Armeria will use a self-signed certificate when there's no keystore configuration.
+    enabled: true
+
+
+armeria:
+  ports:
```

Comment:
Question: Now we do not even need to specify any ports because `server.address` will be picked up, right?

Reply:
Yes, right. We don't need to specify `armeria.*` properties. :-)